### DISCUSSION TITLE
Welcome to Doks Discussions!

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#40] [General]
Hello! This is the place to talk about **Doks** — your documentation sites, themes, and site-building workflow. Ask questions, share tips, or discuss Doks-specific features here.

For questions about the **Thulite framework** or integrations like SEO, images, or plugins, please visit **[Thulite Discussions](https://github.com/orgs/thuliteio/discussions)**.

We’re excited to see what you build with Doks and Thulite! 🌟

**Comment by ImaCrea** [comment]
Hello and thank you very much for this great theme! 

I'm starting a new help center project with it and I'm impressed with the quality of the project and the documentation you wrote with it. Congratulation and all the luck for the future :)

**Comment by gramythedj** [comment]
the theme is awesome, only thing that is missing is this tags n categories for blog posts

**Reply by h-enk** [reply]
Thanks. I will put that on the list!
